utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -35,21 +35,36 @@ THE SOFTWARE.
 #include <omp.h>
 #include <fstream>
 #include <turbojpeg.h>
+#include <boost/filesystem.hpp>
+
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
Please try to use the GPU_SUPPORT flag instead of the __HIP_PLATFORM_HCC__
https://github.com/sampath1117/rpp/blob/sr/test_suite_new/include/rppt_tensor_color_augmentations.h#L69

We are using this flag internally to differentiate between HOST and HIP
Check if this works

Reply:
Done
#ifdef GPU_SUPPORT is working without any errors